src/electionguard/decryption.py

```diff
@@ -454,6 +463,7 @@ def compensate_decrypt(
     missing_guardian_backup: ElectionPartialKeyBackup,
     ciphertext: ElGamalCiphertext,
     extended_base_hash: ElementModQ,
+    guardian_keys: ElectionKeyPair,
```

Comment:
In `compensate_decrypt()` here and also `verify_election_partial_key_backup()` (key_ceremony.py:246), the methods now require a private key of the receiving guardian in order to decrypt the backup.  These signature changes cascade into a lot of other methods and result in some churn in this PR.

Reply:
The debate can still be made to remove the idea of the backup and simply just pass in the `coordinate`.

I think long term this is the simplest API.
```
def compensate_decrypt(
    missing_guardian_coordinate: ElementModQ,
```

Reply:
This is a strong debate to consider going forward with **ElectionGuard** and the future. 

What would be the ideal information to pass into the method?
```py
# Existing
def compensated_decrypt(ciphertext, missing_guardian_backup, extended_base_hash, guardian_keys):
     ...

# Refactored
def decrypt_with_threshold(encrypted_message, key_share, encryption_seed, key_pair):
     ...
```